src/Configuration/AbstractConfiguration.php

```diff
@@ -21,13 +21,22 @@
 
 abstract class AbstractConfiguration implements ConfigurationInterface
 {
-    /** @var string[] */
+    /**
+     * @var string[]
+     * @psalm-var list<string>
```

Comment:
Are you sure you need both here? My understanding is that `@var string[]` is interpreted by Psalm as `list<string>` already...

Reply:
gonna re-check that. I am pretty sure I'd to add it due to a psalm error

Reply:
I believe `string[]` is interpreted as `array<array-key, string>`; the concept of `list<string>` is narrower. If Psalm is not sure if something is truly a list, you can usually wrap it in an `array_values(..)` call

Reply:
Yep, have to add this and I was just too lazy to drop the `@var` annotation. Did that now.